src/workers/index.ts

```diff
@@ -64,6 +66,8 @@ export const workers: Worker[] = [
   postCommentedSlackMessage,
   postUpvotedRep,
   postUpvoteCanceledRep,
+  postDownvotedRep,
+  postDownvoteCanceledRep,
```

Comment:
And maybe also add `should be registered` into tests so we make sure they are exported, as in `collectionUpdated` worker for example.